rasa/core/test.py

```diff
@@ -586,6 +589,9 @@ def plot_story_evaluation(
 
     cnf_matrix = confusion_matrix(test_y, predictions)
 
+    if no_plot:
```

Comment:
let's do this before the `confusion_matrix` call

Reply:
great catch, thanks!